### ISSUE TITLE
Offline-buildable Source Zip

### ISSUE THREAD

**Issue by dcwhite** [SCIInstitute/SCIRun#902]
Requires #191.

Need a nice way to upload an offline source zip to each release.

**Comment by dcwhite** [comment]
@jessdtate How important is this in 2019? We just keep adding more external libraries...

**Comment by jessdtate** [comment]
I don't think this needs to be implemented in 2019.  We can revisit, probably post beta and see if this still makes sense.

**Comment by dcwhite** [comment]
**Triage 2026-06:** Offline-buildable source zip — related to #191. The Superbuild now downloads tarballs but no offline bundle is produced for releases. GitHub Actions could be extended to produce an offline source archive as a release artifact. Low-medium effort to add a CI step.
